### DISCUSSION TITLE
AWS 사용 서비스

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#968] [이야기해봐요]
기존 EC2로 사용 중인 서비스들을 프리티어로 변경에 대해 논의한다. 

RDS, ALB ,,,

**Comment by jminkkk** [comment]
# RDS

운영환경 DB 를 EC2로 운영 중인데 WAS 인스턴스만으로도 프리티어 사용량이 넘기 때문에 RDS 프리티어로 사용하자

**Comment by jminkkk** [comment]
# ELB

## 하자

가장 큰 이유: SSL 인증서, 로드 밸런싱

> 우려) SSL 인증서는 리눅스 cron 설정으로 가능하다

- CICD 방식을 CodeDeploy로 변경 시 리눅스 cron도 관리 대상이다.
- Auto Scaling 을 적용하게 될 경우, AWS로 로드밸런싱을 하는 게 더 편리하다.

## 말자

없음

**Comment by zangsu** [comment]
### 프리티어 요금 가이드 (참고용)

[inpa](https://inpa.tistory.com/entry/AWS-%F0%9F%92%B0-%ED%94%84%EB%A6%AC%ED%8B%B0%EC%96%B4-%EC%9A%94%EA%B8%88-%ED%8F%AD%ED%83%84-%EB%B0%A9%EC%A7%80-%F0%9F%92%B8-%EB%AC%B4%EB%A3%8C-%EC%82%AC%EC%9A%A9%EB%9F%89-%EC%A0%95%EB%A6%AC)

**Comment by jminkkk** [comment]
# ElastiCache - AWS로 글로벌 캐시를 쓰자

## 하자

- DB 인스턴스가 현재는 한대이고, 마이그레이션 전보다 사용량이 많이 작다. 
- 인스턴스를 확장할 가능성(= 분산환경이 될 가능성)이 높다. 

## 말자

- 오버엔지니어링이다.
